From 11dfeb73d1f9cdf8fc493251c8616f5bc338acad Mon Sep 17 00:00:00 2001
From: Liang Hao <haohlliang@gmail.com>
Date: Fri, 14 Aug 2026 19:42:35 +0800
Subject: [PATCH 13908/15333] spi: dw: Remove shadowed dws in dw_spi_setup()

The controller private data is already fetched at the start of
dw_spi_setup(). Drop the redundant inner declaration that shadows it.

Signed-off-by: Liang Hao <haohlliang@gmail.com>
Link: https://patch.msgid.link/20260814114235.31281-1-haohlliang@gmail.com
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 drivers/spi/spi-dw-core.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/spi/spi-dw-core.c b/drivers/spi/spi-dw-core.c
index dfce2fa20adf..206d3f9dd83d 100644
--- a/drivers/spi/spi-dw-core.c
+++ b/drivers/spi/spi-dw-core.c
@@ -1098,7 +1098,6 @@ static int dw_spi_setup(struct spi_device *spi)
 	/* Only alloc on first setup */
 	chip = spi_get_ctldata(spi);
 	if (!chip) {
-		struct dw_spi *dws = spi_controller_get_devdata(spi->controller);
 		u32 rx_sample_dly_ns;
 
 		chip = kzalloc_obj(*chip);
-- 
2.55.0

